src/data-workspace/data-details-sidebar/basic-information.test.js

```diff
@@ -1,19 +1,27 @@
 import { render } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
-import moment from 'moment'
 import React from 'react'
 import BasicInformation from './basic-information.js'
 
 const noop = () => {}
 
 describe('<BasicInformation />', () => {
+    const lastUpdated = '2022-06-28T14:51:14.435'
+    beforeEach(() => {
+        jest.useFakeTimers('modern')
+        // set a fake time (note time zone is UTC+1 while API returns UTC)
+        jest.setSystemTime(new Date('2022-06-28T15:52+01:00').getTime())
+    })
+    afterEach(() => {
+        jest.useRealTimers()
+    })
     it('renders the item name in a heading', () => {
         const item = {
             categoryOptionCombo: 'coc-1',
             comment: null,
             dataElement: 'item-1',
             name: 'Item name',
-            lastUpdated: new Date().toISOString(),
+            lastUpdated,
```

Comment:
This is just a suggestion and will leave it up to you and your preference. I'd prefer to duplicate the string in the mock data instead of extracting it into a variable.

---

Of course it's not bad/wrong whatsoever and subject to opinion. IMO this decreases readability of tests as you'll have to scroll to / remember the value when using tests as documentation. And I think the documentation-like role of tests is more important than reducing the amount of code / repetition.